starlark/testdata/dict.star

```diff
@@ -54,6 +54,14 @@ assert.eq(small.keys(), ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k"])
 
 # duplicate keys are not permitted in dictionary expressions (see b/35698444).
 assert.fails(lambda: {"aa": 1, "bb": 2, "cc": 3, "bb": 4}, 'duplicate key: "bb"')
+# nor in keyword args to dict built-in
+assert.fails(lambda: dict(a=1, a=1), 'dict: duplicate keyword arg: "a"')
+# check that even with many positional args, keyword collisions are detected
+assert.fails(lambda: dict((['b', 3], ), a=4, a=5), 'dict: duplicate keyword arg: "a"')
+assert.fails(lambda: dict((['a',2], ['b', 3]), a=4, a=5), 'dict: duplicate keyword arg: "a"')
```

Comment:
space after comma

Reply:
starlarkfmt, starlarkfmt, my kingdom for an automated code formatter.

Reply:
https://github.com/bazelbuild/buildtools/tree/master/buildifier

At least within Google's configuration, there's an Emacs mode for it and every save of a *.bzl file formats it. But I haven't set it up for *.star files, and I'm not keen to reformat all the test files: some of them are deliberately syntactically invalid, and buildifier is rather more dogmatic than gofmt.